src/providers/twitch/IrcMessageHandler.cpp

```diff
@@ -783,7 +783,15 @@ void IrcMessageHandler::parseUserNoticeMessageInto(Communi::IrcMessage *message,
                 parseTagString(messageText), tags,
                 calculateMessageTime(message).time(), channel);
 
-            msg->flags.set(MessageFlag::Subscription);
+            if (msgType == "viewermilestone")
```

Comment:
This is always false.

Reply:
Do you want to fix this & add a unit test in a follow-up PR @devPoland? If not we can do it later :)